src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -1264,6 +1268,44 @@ public void accessPermittedOnlyFromCurrentBuild() throws Throwable {
             r.buildAndAssertSuccess(main);
         });
     }
+
+    @Ignore("TODO safe fix still TBD")
+    @Test public void placeholderTaskInQueueButAssociatedBuildComplete() throws Throwable {
+        AtomicReference<File> rootDir = new AtomicReference<>();
```

Comment:
Unrelated, but this could be simplified if we exposed `JenkinsSessionRule.home` via a getter. Any thoughts on that? Back when we only had `RestartableJenkinsRule` this kind of test would've been more awkward to write, but now that we can do things to `JENKINS_HOME` between restarts with `JenkinsSessionsRule` it's pretty straightforward.

Reply:
Funny you should mention that: https://github.com/jenkinsci/jenkins-test-harness/pull/352#discussion_r760545577

Reply:
See 3abef26 (also switched to `TemporaryFolder` and removed a `System.out.println` that was not meant to be committed).